### ISSUE TITLE
Notice and GlobalNotice

### ISSUE THREAD

**Issue by outdoor2kode** [WordPress/wporg-two-factor#208]
Currently, we have two types of notices: `Notice` and `GlobalNotice`.

A `Notice` is a prompt box that appears in various positions within each component. It could be an error notice, or a warning, info, success notice, etc.

`GlobalNotice`, on the other hand, is a black prompt box that pops up in a fixed position in the top right corner. Its message is set through `setGlobalNotice`.

We could consider consolidating these two to enhance readability.

**Comment by iandunn** [comment]
They seem to serve different purposes to me, but maybe that's just mainly CSS?

`GlobalNotice` is more like a `Snackbar`, so maybe we should use that component instead? Offhand I don't remember why we wrote a custom one.

**Comment by outdoor2kode** [comment]
Yeah, that's a good point. Originally, I thought there were probably some reasons for naming it as `GlobalNotice` so I figured it could reduce confusion by merging the two components. But it looks like we could directly replace `GlobalNotice` with `Snackbar`, or these `GlobalNotice` and `Notice` naming would be likely to cause some misunderstanding for someone new to the code or when we come back to look at the code after a while.
